src/index.js

```diff
@@ -183,7 +178,7 @@ module.exports = {
         closeServer()
       }
 
-      processCypressResults(results, failPlugin)
+      processCypressResults(results, args.utils.build)
```

Comment:
CI caught an error - it should be `arg`, right?

Reply:
Yes, fixed! :+1: